package.json

```diff
@@ -20,24 +20,30 @@
     "graphql-yoga": "^1.16.7",
     "inquirer": "^6.2.1",
     "jest-coverage-badges": "^1.1.2",
+    "jshint": "^2.10.1",
     "minimist": "^1.2.0",
     "node-fetch": "^2.3.0",
     "touch": "^3.1.0"
   },
   "devDependencies": {
+    "eslint": "^5.13.0",
     "jest": "^24.0.0"
   },
   "jest": {
     "coverageReporters": [
       "json-summary",
       "text",
       "lcov"
+    ],
+    "modulePathIgnorePatterns": [
+      "coverage"
     ]
   },
   "scripts": {
     "test": "jest --verbose",
     "test:coverage": "npm test -- --coverage",
-    "test:badges": "npm run test:coverage  && jest-coverage-badges"
+    "test:badges": "npm run test:coverage  && jest-coverage-badges",
+    "lint": "eslint"
```

Comment:
this needs to be "lint":"eslint ."

to point eslint to start int he current directory